spec/factories/ex_links.rb

```diff
@@ -0,0 +1,12 @@
+FactoryGirl.define do
+  factory :ex_link do
+    label "MyString"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.